test/sqllogictest/chbench.slt

```diff
@@ -300,23 +300,19 @@ ORDER BY n_name, su_name, i_id
 
 %6 =
 | Get materialize.public.supplier (u34)
-| ArrangeBy (#0) (#3)
+| ArrangeBy (#3)
 
 %7 =
 | Get materialize.public.nation (u31)
-| ArrangeBy (#0) (#2)
+| ArrangeBy (#0)
 
 %8 =
 | Get materialize.public.region (u37)
 | ArrangeBy (#0)
 
 %9 =
-| Join %5 %6 %7 %8 (= #17 #18) (= #21 #25) (= #27 #29)
-| | implementation = DeltaQuery
```

Comment:
Is this ok? I don't know anything besides I think DeltaQuery is good?

Reply:
ty for pointing this out. it's a known issue (#4171), so I simply changed the table's column type to express the type that it used to resolve to (`integer`).